app/views/docs/functions.phtml

```diff
@@ -19,36 +19,36 @@ $runtimes = $this->getParam('runtimes', []);
 
 <p>You can deploy your code using the <a href="/docs/command-line">Appwrite CLI</a> tool, the Appwrite <a href="/docs/getting-started-for-server">Server API</a> or manually from the Appwrite console.</p>
 
-<h3><a href="/docs/functions#createTag" id="createTag">Create Tag</a></h3>
+<h3><a href="/docs/functions#createDeployment" id="createDeployment">Create Deployment</a></h3>
 
 <ul class="phases clear" data-ui-phases>
     <li>
         <h3>Unix</h3>
 
         <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-            <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite functions createTag \
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite functions createDeployment \
     --functionId=6012cc93d5a7b \
-    --command="node index.js" \
+    --entrypoint="index.js" \
     --code="/myrepo/myfunction"</code></pre>
         </div>
     </li>
     <li>
         <h3>CMD</h3>
 
         <div class="ide margin-bottom" data-lang="bash" data-lang-label="CMD">
-            <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite functions createTag ^
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite functions createDeployment ^
     --functionId=6012cc93d5a7b ^
-    --command="node index.js" ^
+    --entrypoint="index.js" ^
     --code="/myrepo/myfunction"</code></pre>
```

Comment:
I don't think this will work anymore.. I think you MUST be in the repository where you are tarring the file, otherwise build will not find files properly. We should try this with new CLI to make sure.